### ISSUE TITLE
DiCE Explanations for Autoencoder model having Time series Data

### ISSUE THREAD

**Issue by iuush** [interpretml/DiCE#275]

**Comment by gaugup** [comment]
@iuush could you provide more description as to what you are looking for?

Regards,

**Comment by iuush** [comment]
Dear Gaurav,

I am using an Autoencoder model for finding the anomaly in the tabular
data. But your package is not working for my model and code.

PFA my code(Notebook).

**Comment by iuush** [comment]
Dear Gaurav,

I need it urgently. Could you please help me ASAP.

Thanks
Ayush
+91-9528553052

**Comment by iuush** [comment]
Dear Gaurav,

I am using an Autoencoder model for finding the anomaly in the tabular
data. But your package is not working for my model and code.

PFA my Model()

**Comment by iuush** [comment]
Dear Gaurav,

I need it urgently. Could you please help me ASAP.

Thanks
Ayush
+91-9528553052

**Comment by amit-sharma** [comment]
@iuush I looked through your messages above and did not find the attached notebook or model. Can you reshare it? 

We need a minimum reproducible code to understand the issue.

**Comment by iuush** [comment]
Dear Interpretml/Dice team,

This model is not working for a LSTM autoencoder model or any autoencoder
model.

My autoencoder model is:

 # create a model by subclassing Model class in tensorflow
class AutoEncoder(Model):
  """
  Parameters
  ----------
  output_units: int
    Number of output units

  code_size: int
    Number of units in bottle neck
  """

  def __init__(self, output_units, code_size=4):
    super().__init__()
    self.encoder = Sequential([
      Dense(64, activation='relu'),
      Dropout(0.1),
      Dense(32, activation='relu'),
      Dropout(0.1),
      Dense(16, activation='relu'),
      Dropout(0.1),
      Dense(code_size, activation='relu')
    ])
    self.decoder = Sequential([
      Dense(16, activation='relu'),
      Dropout(0.1),
      Dense(32, activation='relu'),
      Dropout(0.1),
      Dense(64, activation='relu'),
      Dropout(0.1),
      Dense(output_units, activation='sigmoid')
    ])

  def call(self, inputs):
    encoded = self.encoder(inputs)
    decoded = self.decoder(encoded)
    return decoded

  def predict_proba(self, inputs):
    encoded = self.encoder(inputs)
    decoded = self.decoder(encoded)
    return 0.1

model = AutoEncoder(output_units=X_train_s.shape[1])

model.compile(loss='mse',optimizer='adam')

X_pred = model.predict(df_s)

regards
Ayush

**Comment by iuush** [comment]
Dear Gaurav,

Could you please share your contact number. It would be very helpful.

Thanks
Ayush
